docs/book/src/forc/plugins/forc_client/index.md

```diff
@@ -1,26 +1,76 @@
+## Option 1: Sign transactions via forc-client using your local forc-wallet vault
 
-## Signing transactions using `forc-wallet` CLI
+If you used forc-wallet before, a vault which securely holds your private key is created and written to your file-system in a password encrypted format. forc-client is compatible with forc-wallet such that it can read that vault by asking you your password and use your account to sign transactions.
 
-To submit the transactions created by `forc deploy` or `forc run`, you need to sign them first (unless you are using a client without UTXO validation). To sign a transaction you can use `forc-wallet` CLI. This section is going to walk you through the whole signing process.
+Example:
 
-By default `fuel-core` runs without UTXO validation, this allows you to send invalid inputs to emulate different conditions.
+```console
+> forc deploy
 
-If you want to run `fuel-core` with UTXO validation, you can pass `--utxo-validation` to `fuel-core run`.
+  Building /Users/test/test-projects/test-contract
+  Finished release [optimized + fuel] target(s) in 11.39s
 
-To install `forc-wallet` please refer to `forc-wallet`'s [GitHub repo](https://github.com/FuelLabs/forc-wallet#forc-wallet).
+Please provide the password of your encrypted wallet vault at "/Users/ceylinbormali/.fuel/wallets/.wallet":
+   Deploying contract: impl-contract
 
-1. Construct the transaction by using either `forc deploy` or `forc run`. To do so simply run `forc deploy` or `forc run` with your desired parameters. For a list of parameters please refer to the [forc-deploy](./forc_deploy.md) or [forc-run](./forc_run.md) section of the book. Once you run either command you will be asked the address of the wallet you are going to be signing with. After the address is given the transaction will be generated and you will be given a transaction ID. At this point CLI will actively wait for you to insert the signature.
+---------------------------------------------------------------------------
+Account 0: fuel12pls73y9hnqdqthvduy2x44x48zt8s50pkerf32kq26f2afeqdwq6rj9ar
+
+Asset ID : f8f8b6283d7fa5b672b530cbb84fcccb4ff8dc40f8176ef4544ddb1f1952ad07
+Amount   : 2197245
+---------------------------------------------------------------------------
+
+Please provide the index of account to use for signing: 0
+Do you agree to sign this transaction with fuel12pls73y9hnqdqthvduy2x44x48zt8s50pkerf32kq26f2afeqdwq6rj9ar? [y/N]: y
+
+
+Contract impl-contract Deployed!
+
+Network: https://testnet.fuel.network
+Contract ID: 0x94b712901f04332682d14c998a5fc5a078ed15321438f46d58d0383200cde43d
+Deployed in block 00114d4d
+```
+
+As it can be seen from the example, forc-client asks for your password to decrypt the forc-wallet vault, and list your accounts so that you can select the one you want to fund the transaction with.
```

Comment:
```suggestion
As it can be seen from the example, `forc-client` asks for your password to decrypt the `forc-wallet` vault, and list your accounts so that you can select the one you want to fund the transaction with.
```